mobile/src/main/java/de/michelinside/glucodatahandler/widget/GlucoseBaseWidget.kt

```diff
@@ -241,6 +245,37 @@ abstract class GlucoseBaseWidget(private val type: WidgetType,
             else
                 remoteViews.setViewVisibility(R.id.cobText, View.VISIBLE)
         }
+
+
+        if (hasBatteryLevel) {
+            var deviceNameStr = "No WearOS Device"
```

Comment:
No static text! Check, if there is already one. For example "activity_main_disconnected_label".

Reply:
My bad, will add new string